mico-core/src/main/java/io/github/ust/mico/core/dto/MicoServiceDependencyGraphEdgeDTO.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.github.ust.mico.core.dto;
+
+import io.github.ust.mico.core.model.MicoService;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Accessors(chain = true)
+public class MicoServiceDependencyGraphEdgeDTO {
+
+
+    private String sourceShortName;
+    private String sourceVersion;
+    private String targetShortName;
+    private String targetVersion;
```

Comment:
We could reduce these parameters when making use of the `id` instead of `shortName` and `version`.
```suggestion
    private Long sourceId;
    private Long targetId;
```

Reply:
I thought the use of the Id is discouraged in favor of the shortName and version.

Reply:
There is a discussion to remove ids from all api results already so it would be unwise to add a result that relies on these ids.

Reply:
Why the hell should we want to remove the `id`s?

Reply:
Only for the UI the ids would disappear/be hidden. In the backend the ids would still be used.

Reply:
But nevertheless, I don't see any benefit if we hide the `id`s.

Reply:
There is no real benefit for us to hide the ids but for consistency this should still not use ids.